root/utility/formatUserDate.js

```diff
@@ -163,11 +163,11 @@ export function formatUserDateObject(
 
   const preferences = $c && $c.user ? $c.user.preferences : null;
   const timezone =
-    (options && options.timezone) ||
-    (preferences && preferences.timezone) ||
+    (options ? options.timezone : null) ||
```

Comment:
Yeah, I guess except returning null instead of undefined, but Flow doesn't like it:

```
Error ┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈ root/utility/formatUserDate.js:166:5

Sketchy null check on string [1] which is potentially an empty string. Perhaps you meant to check for null or
undefined [1]? [sketchy-null-string]

 [1] 152│   timezone?: string,
        :
     163│
     164│   const preferences = $c && $c.user ? $c.user.preferences : null;
     165│   const timezone =
     166│     options?.timezone ||
     167│     preferences?.timezone ||
     168│     'UTC';
     169│   let format =

```

Not sure why it accepts it like I have it, maybe because it's more explicit?